doc/source/serve/doc_code/vllm_openai_example.py

```diff
@@ -111,6 +112,19 @@ def build_app(cli_args: Dict[str, str]) -> serve.Application:
 
     Supported engine arguments: https://docs.vllm.ai/en/latest/models/engine_args.html.
     """  # noqa: E501
+    if "device" in cli_args.keys():
```

Comment:
device -> accelerator to follow the naming convention here - https://docs.ray.io/en/latest/ray-core/scheduling/accelerators.html